client/src/components/FormInput/FormInput.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import './FormInput.sass';
+
+export default class FormInput extends Component {
+    input = () => {
+        if (this.props.inputType === 'little-input') {
+            return <input className={this.props.inputType} placeholder={this.props.inputPlaceholder} />
+        } else {
+            return <textarea className={this.props.inputType} placeholder={this.props.inputPlaceholder} />
+        }
+    }
+    render() {
+        return(
+            <>
+                {this.input()}
```

Comment:
`this.props.inputType === 'little-input' ? <input className={this.props.inputType} placeholder={this.props.inputPlaceholder} /> :<textarea className={this.props.inputType} placeholder={this.props.inputPlaceholder} />`